glide-core/src/client/value_conversion.rs

```diff
@@ -138,12 +137,10 @@ pub(crate) fn convert_to_expected_type(
                         Value::Nil => Ok(Value::Nil),
                         _ => match from_owned_redis_value::<bool>(item.clone()) {
                             Ok(boolean_value) => Ok(Value::Boolean(boolean_value)),
-                            _ => Err((
-                                ErrorKind::TypeError,
-                                "Could not convert value to boolean",
-                                format!("(value was {:?})", item),
-                            )
-                                .into()),
+                            _ => {
+                                Err((ErrorKind::TypeError, "Could not convert value to boolean")
```

Comment:
add type

Reply:
also, why is this (and below) a boolean and not JSON toggle?

Reply:
[`JSON.TOGGLE`](https://redis.io/docs/latest/commands/json.toggle/) returns 0/1 being converted to `bool` or `nil`.
Not a mistake I suppose.